python/setup.py

```diff
@@ -145,61 +151,52 @@ def run(self):
   if cpp_impl in sys.argv:
     sys.argv.remove(cpp_impl)
     # C++ implementation extension
-    ext_module_list.append(Extension(
+    ext_module_list.append(
+      Extension(
         "google.protobuf.pyext._message",
-        [ "google/protobuf/pyext/descriptor.cc",
+        [
+          "google/protobuf/pyext/descriptor.cc",
           "google/protobuf/pyext/descriptor_containers.cc",
           "google/protobuf/pyext/descriptor_pool.cc",
-          "google/protobuf/pyext/message.cc",
           "google/protobuf/pyext/extension_dict.cc",
+          "google/protobuf/pyext/message.cc",
+          "google/protobuf/pyext/repeated_composite_container.cc",
           "google/protobuf/pyext/repeated_scalar_container.cc",
-          "google/protobuf/pyext/repeated_composite_container.cc" ],
+        ],
         define_macros=[('GOOGLE_PROTOBUF_HAS_ONEOF', '1')],
-        include_dirs = [ ".", "../src" ],
-        libraries = [ "protobuf" ],
-        library_dirs = [ '../src/.libs' ],
-        ))
+        include_dirs=[".", "../src"],
+        libraries=['protobuf'],
+        library_dirs=['../src/.libs'],
+      )
+    )
     os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION'] = 'cpp'
 
-  setup(name = 'protobuf',
-        version = GetVersion(),
-        packages = [ 'google' ],
-        namespace_packages = [ 'google' ],
-        test_suite = 'google.protobuf.internal',
-        # Must list modules explicitly so that we don't install tests.
```

Comment:
I believe that's covered by `exclude google/protobuf/internal/*_test.py` in MANIFEST.in